[MIG] hr_timesheet_sheet migration to 17.0

vnikolayev1 opened the pull request:
- ported module from version 16 to version 17
- fixed/updated tests, logic based on views change
- updated pre-commit auto fixes

MiquelRForgeFlow committed:
[11.0][MIG] hr_timesheet_sheet (#125)

* [10.0] hr_timesheet_sheet

* [11.0][MIG] hr_timesheet_sheet

* [REMOVE] hr_timesheet.sheet.account

* [REMOVE] 'new' state

* [ADD] Tests

* [UPD] Adapt to multicompany

* [ADD] Add more tests (include multicompany tests)

* [FIX] project_task_stage_allow_timesheet: show error message only if task

* [ADD] Migration scripts to v11

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

yostashiro committed:
Translated using Weblate (Japanese)

Currently translated at 98.9% (88 of 89 strings)

Translation: hr-timesheet-11.0/hr-timesheet-11.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/hr-timesheet-11-0/hr-timesheet-11-0-hr_timesheet_sheet/ja/

Commit:
Translated using Weblate (Português (Brasil))

Currently translated at 100,0% (89 of 89 strings)

Translation: hr-timesheet-11.0/hr-timesheet-11.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/hr-timesheet-11-0/hr-timesheet-11-0-hr_timesheet_sheet/pt_BR/

JordiBForgeFlow committed:
[11.0][hr_timesheet_sheet]: allow to display more than 40 cells

JordiBForgeFlow committed:
hr_timesheet_sheet: add menu 'Timesheet Sheets to Approve'

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

JordiBForgeFlow committed:
hr_timesheet_sheet: pass selected project context to task creation

astirpe committed:
[11.0][IMP] hr_timesheet_sheet: ease extensions (#131)

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

astirpe committed:
Fix task domain and default project_id

astirpe committed:
Bump version

astirpe committed:
[IMP] hr_timesheet_sheet ux

astirpe committed:
Code review

astirpe committed:
Update contributors list

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

astirpe committed:
[FIX] hr_timesheet_sheet: "Refuse" button name (#134)

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

LoisRForgeFlow committed:
[11.0][IMP] hr_timesheet_sheet: merge hr_timesheet_sheet_week_start_day into this module, which adds a configuration to select the week start day.

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

JordiBForgeFlow committed:
restrict selection of projects to those that have 'allow_timsheets' set

alexey-pelykh committed:
[MIG] hr_timesheet_sheet: Migration to 12.0

pedrobaeza committed:
[FIX] hr_timesheet_sheet: Don't fail tests on Sunday

If you run tests on Sunday, test_4 was not prepared for it, as next day is Monday,
which belongs to other week, and thus not included in same timesheet. With this,
we cover that case, decreasing one day instead of adding it.

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: more extendability & cleanup

MiquelRForgeFlow committed:
[FIX] hr_timesheet_sheet: who approves timesheets should be the manager

AdriaGForgeFlow committed:
[FIX] Fix related fields in hr_timesheet_sheet

bizzappdev committed:
[FIX]hr_timesheet_sheet : fix singleton error in _compute_line_ids

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

astirpe committed:
[FIX] timesheet sheet lines inconsistency

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: texts & formatting

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: unlink only generated AAL

bizzappdev committed:
[FIX]hr_timesheet_sheet : unit amount will display in formate hr:minute

MiquelRForgeFlow committed:
[IMP] Rename method / parameters to a better descriptive name

MiquelRForgeFlow committed:
[FIX] Correctly handle partial amounts of timesheets

astirpe committed:
[FIX] hr_timesheet_sheet: sheet_id when modifying the timesheet

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: protect specific list of fields

MiquelRForgeFlow committed:
[IMP] Make a constant for the '/' name for empty lines

MiquelRForgeFlow committed:
[FIX] Avoid cleaning when set to draft

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-timesheet-12.0/hr-timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/hr-timesheet-12-0/hr-timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
hr_timesheet_sheet: version bump

astirpe committed:
[FIX] Wrong decorator

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-timesheet-12.0/hr-timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/hr-timesheet-12-0/hr-timesheet-12-0-hr_timesheet_sheet/

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (142 of 142 strings)

Translation: hr-timesheet-12.0/hr-timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/hr-timesheet-12-0/hr-timesheet-12-0-hr_timesheet_sheet/nl/

astirpe committed:
[ADD] Constraint company_id in timesheet sheet line

astirpe committed:
[ADD] company_id in blacklist of timesheet sheet line

astirpe committed:
[REM] Obsolete constraint

astirpe committed:
[IMP] hr_timesheet_sheet: fix amount calculation issues + code simplification (#211)

* [IMP] Fix amount calculation issues + code simplification

* [FIX] Fix employee_id related stuff

* [IMP] Ease customizations of message_subscribe_users

* [IMP] List view easier to read

* [IMP] Extra code simplification

* [IMP] Make use of batch create

* [FIX] Update company when onchange employee_id

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: sheet name

astirpe committed:
[FIX] Allow timesheet sheet when employee's company is not set

astirpe committed:
[FIX] Consistency of company on projects and employees

astirpe committed:
[FIX] Add lines of other employees

astirpe committed:
[12.0][FIX] hr_timesheet_sheet: subscribe user when confirm (#227)

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

MiquelRForgeFlow committed:
[FIX] Don't create analytic lines in onchange

And other minor improvements. The way to handle it is to add a new transient model to correctly assure you only create analytic lines when saving.

MiquelRForgeFlow committed:
[FIX] Tests

tarteo committed:
[IMP] hr_timesheet_sheet: Clearer interface

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: tests

OCA-git-bot committed:
[UPD] README.rst

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: consistency with v11

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: review policy

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: more hooks

marylla committed:
Translated using Weblate (German)

Currently translated at 32.2% (48 of 149 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/de/

astirpe committed:
[FIX] Expected singleton

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

astirpe committed:
[12.0][FIX] Do not remove lines when removing the timesheet sheet

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (149 of 149 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (149 of 149 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/de/

britoederr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (155 of 155 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/pt_BR/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: remove non-existent models

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: error details

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
[ADD] hr_timesheet_sheet: policies

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

chienandalu committed:
Translated using Weblate (Spanish)

Currently translated at 93.6% (161 of 172 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/es/

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 75.0% (129 of 172 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/it/

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: allow employee to review own non-submitted timesheets

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (172 of 172 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (176 of 176 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/de/

astirpe committed:
[FIX] hr_timesheet_sheet_policy*: admin always reviewer

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: don't store possible_reviewer_ids

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: use unique IDs in line generation

astirpe committed:
[FIX] Clean/merge timesheets when in draft only

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: same week in different years is still same

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

pedrobaeza committed:
Translated using Weblate (Spanish)

Currently translated at 91.5% (161 of 176 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/es/

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: hooks

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: add id to allow modules add options

alexis-via committed:
hr_timesheet_sheet: fix FR translation that trigger a crash

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: add kanban view

Commit:
Translated using Weblate (Italian)

Currently translated at 97.2% (171 of 176 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/it/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/

alexey-pelykh committed:
[IMP] hr_timesheet_sheet: _get_subscribers hook

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: lost kanban mode

Commit:
[IMP] : black, isort

Commit:
[MIG] hr_timesheet_sheet: Migration to 13.0

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet/nl/

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: drop fuzzy+python-format translations

MiquelRForgeFlow committed:
[MIG] hr_timesheet_sheet: Migration to 13.0 (complete)

MiquelRForgeFlow committed:
[IMP] hr_timesheet_sheet: black, isort, prettier

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-13.0/timesheet-13.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-13-0/timesheet-13-0-hr_timesheet_sheet/

alexey-pelykh committed:
[FIX] hr_timesheet_sheet: openupgrade

Commit:
[FIX] failing tests, linting

OCA-git-bot committed:
hr_timesheet_sheet 13.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-13.0/timesheet-13.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-13-0/timesheet-13-0-hr_timesheet_sheet/

pedrobaeza committed:
[FIX] hr_timesheet_sheet: Add review policies for other groups

Apart from the existing one, let for "Employees/Officer" (and clarified in the review
text), 2 new review policies are added:

* For HR managers
* For timesheet managers

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-13.0/timesheet-13.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-13-0/timesheet-13-0-hr_timesheet_sheet/

pedrobaeza committed:
Translated using Weblate (French)

Currently translated at 23.6% (42 of 178 strings)

Translation: timesheet-13.0/timesheet-13.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-13-0/timesheet-13-0-hr_timesheet_sheet/fr/

vishalaktiv committed:
[FIX] hr_timesheet_sheet  WARNING DB odoo.osv.expression: The domain term '('id', '=', [ID1, ID2,..., IDn])' should use the 'in' or 'not in' operator.

alexey-pelykh committed:
[UPD] Brainbean Apps => CorporateHub

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

OCA-git-bot committed:
hr_timesheet_sheet 13.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-13.0/timesheet-13.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-13-0/timesheet-13-0-hr_timesheet_sheet/

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

ypapouin committed:
[FIX] Typo

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

OCA-git-bot committed:
hr_timesheet_sheet 13.0.1.1.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-13.0/timesheet-13.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-13-0/timesheet-13-0-hr_timesheet_sheet/

tarteo committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (180 of 180 strings)

Translation: timesheet-13.0/timesheet-13.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-13-0/timesheet-13-0-hr_timesheet_sheet/nl/

schhatbar-initos committed:
[IMP] hr_timesheet_sheet: black, isort, prettier

schhatbar-initos committed:
[14.0][MIG]hr_timesheet_sheet

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

OCA-git-bot committed:
[UPD] README.rst

Yvesldff committed:
Added translation using Weblate (French (France))

Yvesldff committed:
Translated using Weblate (French (France))

Currently translated at 76.2% (138 of 181 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/fr_FR/

Yvesldff committed:
Translated using Weblate (French (France))

Currently translated at 82.3% (149 of 181 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/fr_FR/

OSevangelist committed:
Translated using Weblate (German)

Currently translated at 90.0% (162 of 180 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/de/

Yvesldff committed:
Translated using Weblate (French)

Currently translated at 80.0% (144 of 180 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/fr/

Commit:
Do not try to remove invoiced lines

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/

innovara committed:
[14.0] [FIX] hr_timesheet_sheet Week %s translation

OCA-git-bot committed:
hr_timesheet_sheet 14.0.1.0.1

oca-travis committed:
[UPD] Update hr_timesheet_sheet.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/

tarbib committed:
[IMP] hr_timesheet_sheet: Screenshot added for quick understanding

Commit:
Do not merge invoiced timesheet lines

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
hr_timesheet_sheet 14.0.1.0.2

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 47.8% (87 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 61.5% (112 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 88.4% (161 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 88.4% (161 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 94.5% (172 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 97.2% (177 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/sv/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 98.3% (179 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/nl/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 98.3% (179 of 182 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet/nl/

OCA-git-bot committed:
hr_timesheet_sheet 14.0.1.0.3

OCA-git-bot committed:
hr_timesheet_sheet 14.0.1.0.4

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

victoralmau committed:
[FIX] hr_timesheet_sheet: Add sale_timesheet dependency (timesheet_invoice_id field of account.analytic.line adds it).
Tests currently fail without having sale_timesheet addon installed.
TT35948

OCA-git-bot committed:
hr_timesheet_sheet 14.0.1.1.0

CRogos committed:
[IMP] hr_timesheet_sheet: black, isort, prettier

CRogos committed:
[MIG] hr_timesheet_sheet: Migration to 15.0

carmenbianca committed:
[IMP] hr_timesheet_sheet: timesheet_sheet_ids on hr.employee

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>
Signed-off-by: Carmen Bianca BAKKER <carmen@coopiteasy.be>

Commit:
Fix python-format translation issues.

Commit:
[UPD] Update hr_timesheet_sheet.pot

OCA-git-bot committed:
[UPD] README.rst

vincent-hatakeyama committed:
Translated using Weblate (French)

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/fr/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 77.4% (141 of 182 strings)

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/pt_BR/

Commit:
[UPD] Update hr_timesheet_sheet.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 81.9% (150 of 183 strings)

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/it/

Commit:
[UPD] Update hr_timesheet_sheet.pot

OCA-git-bot committed:
hr_timesheet_sheet 15.0.1.1.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-15.0/timesheet-15.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-hr_timesheet_sheet/

tarteo committed:
[MIG] hr_timesheet_sheet: Migration to 16.0

[MIG] hr_timesheet_sheet: Migration to 16.0

[MIG] hr_timesheet_sheet: Migration to 16.0

[MIG] hr_timesheet_sheet: Migration to 15.0

[MIG] hr_timesheet_sheet: Migration to 16.0

[MIG] hr_timesheet_sheet: Migration to 16.0

[MIG] hr_timesheet_sheet: Migration to 16.0

[DON'T MERGE] test-requirements.txt

fixup

fixup

fixup

[FIX] 'Add new line' Button width

tarteo committed:
Fix for https://github.com/odoo/odoo/pull/120597

Commit:
[UPD] Update hr_timesheet_sheet.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/es/

Commit:
[UPD] Update hr_timesheet_sheet.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

OCA-git-bot committed:
[UPD] README.rst

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/es/

Commit:
[UPD] Update hr_timesheet_sheet.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

MiquelRForgeFlow committed:
[FIX] hr_timesheet_sheet: `image_128` -> `avatar_128`

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/pt_BR/

remi-filament committed:
Translated using Weblate (French)

Currently translated at 95.0% (173 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/fr/

Commit:
[UPD] Update hr_timesheet_sheet.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

pere-aquarian committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/ca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 99.4% (180 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/sv/

LauraCForgeFlow committed:
[16.0][FIX] hr_timesheet_sheet: permissions on access to menus

LauraCForgeFlow committed:
[16.0][FIX] hr_timesheet_sheet: update translations (Spanish)

Commit:
[UPD] Update hr_timesheet_sheet.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 82.5% (151 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 84.1% (154 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/pt/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/pt_BR/

pedrobaeza commented:
Thanks for the contribution.

Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0.

If the jump is between several versions, you have to modify the source branch in the main command to accommodate it to this circumstance.

Commit:
[IMP] hr_timesheet_sheet: pre-commit auto fixes

Commit:
[MIG] hr_timesheet_sheet migration to 17.0

- ported module from version 16 to version 17
- fixed/updated tests, python logic based on views changes
- fixed linter errors

vnikolayev1 commented:
@pedrobaeza Thank you for migration guide, here is an update.

pedrobaeza commented:
/ocabot migration hr_timesheet_sheet

pedrobaeza commented:
@vnikolayev1 it seems there's another migration on progress. Did you check it?

vnikolayev1 commented:
@pedrobaeza probably i should check if it exists first.

Looks like it has some extra functionality here ( _check_can_write method) https://github.com/OCA/timesheet/pull/687/commits/e9527413b3a9d567f73ea267aa2c1cf9fd4b7cdb
settings views are not updated, 
tests does not look like were touched either (they should be updated because we change readonly logic)

Sadly they did same mistake like i did on my first push - did not use patch so it' s hard to see everything what is changed

Looks like i did a bit better porting overall, so maybe their extra changes can be placed over mine later

p.s. is there anything i could read on how to check existing MR and what to do with them(or what should i google)? i am relatively new to this.

pedrobaeza commented:
Please review and put comments in the other PR.

pedrobaeza commented:
Please include #712 if this PR is finally used.
